README.md

```diff
@@ -36,19 +40,26 @@ nanoff --target M5StickCPlus --update --preview --serialport COM3 --baud 115200
 # For the M5Core2:
 nanoff --target M5Core2 --update --preview --serialport COM3
 ```
+NOTE3: If the nanoff commands fail, make sure you did what is written on NOTE1.
 
 Once you have the nugets, you can then enjoy accessing the screen, the accelerometer, get a Grove I2C connecter, add events on the buttons. And you don't even need to think about anything, all is done for you in the most transparent way!
 
 > Note: All the classes that you'll have access are all using the Lazy pattern to be instantiated including the screen. This have the advantage to use as little memory and setup time as possible.
 
 In the samples below, we'll use either M5Stack, either M5Stick as examples, they are all working in a very similar way.
 
+### namespaces
+
+Make sure you add the proper namespaces reference to your C# program header e.g.
+using nanoFramework;
+
 ### Screen
 
-The only thing you need to do to access the screen is to initialize it:
+The only thing you need to do to access the screen is to initialize it; nevetheless this is also target specific e.g.:
```

Comment:
```suggestion
The only thing you need to do to access the screen is to initialize it (please note that `InitializeScreen()` it's platform specific) e.g.:
```

Reply:
in this one i don't actually agree, it is not a platform, it is actually an specific target within a family of products , maybe of the same platform....if you call platform to all M5 Stack (company) products and e..g another platform the STM32 family of products....again, here it is target specific , each M5Stack target needs a different screen initialization semantics ...(which is not nice actually)...

Reply:
Yes, that should be target, not platform.